examples/training/object_detection/pascal_voc/retina_net.py

```diff
@@ -38,9 +60,38 @@
 ]
 class_mapping = dict(zip(range(len(class_ids)), class_ids))
 
+flags.DEFINE_string(
+    "weights_path",
+    "weights_{epoch:02d}.h5",
+    "Directory which will be used to store weight checkpoints.",
```

Comment:
h5 seems to be a file, not a directory.

Reply:
Changing to `weights_name` (compare to file_name)